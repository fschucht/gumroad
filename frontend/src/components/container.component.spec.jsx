import renderer from 'react-test-renderer';
import { Container } from './container.component';

describe('Container', () => {
  it('should render a Container component', () => {
    const renderedComponent = renderer.create(<Container>Hello World!</Container>).toJSON()

    expect(renderedComponent).toMatchSnapshot()
  })
})