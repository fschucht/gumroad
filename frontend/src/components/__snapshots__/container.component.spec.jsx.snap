// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container should render a Container component 1`] = `
<div
  className="container"
>
  Hello World!
</div>
`;
